Tests/Tests/Saml2AssertionExtensionsTests.cs

```diff
@@ -183,5 +183,61 @@ public void Saml2AssertionExtensions_ToXElement_Statements()
 
             actualAuthnXml.Should().BeEquivalentTo(expectedAuthnXml);
         }
+
+        [TestMethod]
+        public void Saml2AssertionExtensions_ToXElement_Statements_Xmlns()
```

Comment:
Clean up this test and make sure it only test the XS declaration. The authnInstant checks are not relevant for this test. I guess that's something that's left from a test you copied?